libraries/AP_DDS/AP_DDS_config.h

```diff
@@ -134,6 +134,10 @@
 #define AP_DDS_MODE_SWITCH_SERVER_ENABLED 1
 #endif
 
+#ifndef AP_DDS_COPTER_TAKEOFF_SERVER_ENABLED
+#define AP_DDS_COPTER_TAKEOFF_SERVER_ENABLED 1
```

Comment:
Not a blocker, but maybe we should only enable this if we're on copter?

Reply:
The dev team wants us to change the name to AP_DDS_VTOL_TAKEOFF_SERVER_ENABLED because it works on quadplane too. Is that ok? Otherwise looks good.

Reply:
Got it Ryan! I’ll make the change.